src/Raven.Client/Constants.cs

```diff
@@ -241,6 +241,8 @@ private Fields()
 
                     public const string ReduceKeyValueFieldName = "key()";
 
+                    public const string ValueFieldName = "value()";
```

Comment:
I'm not sure that I'm following, what are you storing under this field name?

Reply:
Result of Map function for Map TS indexes.

Reply:
Why is that needed? Aren't we outputting the fields as usual here?

Reply:
This is how we are handling that in Map-Reduce and since Map TS is behaving similar to Map-Reduce (qyerying-wise). I;m storing outcome same way as there. The returned query results from Map TS are exactly the values from the index, not the TS entries.

Reply:
I see, make sense, then.